### PR TITLE
[Topic] - Favorite kind of chocolate

### PR DESCRIPTION
**Pull request by lunulae**
I wanted to try to squeeze in this one before Valentine's Day, just a small topic about candy for the season. 🥰 

edit; Not 100% sure on the category on this one. I thought about making it Holidays but I think it's more generally applicable.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -16300,3 +16300,30 @@ label monika_gmos:
...
+    )
+
+label monika_favchocolate:
+    m 2hkb "Ooh, that's a tough question!"
+    m 2euu "I think if I had to pick, it would be dark chocolate."
+    m 3eub "It contains very little or no milk, so it has a less creamy texture, but a nice bittersweet taste."
+    m 3hub "Not to mention, it's rich in antioxidants and can even give some cardiovascular benefits! {w=0.3}{nw}"
+    extend 3hksdlb "...In moderation, of course."
+    m 1hubla "The taste kind of reminds me of a mocha coffee. Maybe the similarity in flavors is why I like it most."
+    m 1eud "Speaking of coffee, though...if I were drinking some, I think I might prefer milk or white chocolate for balance."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy:16322]: We should probably probably check if she's currently drinking coffee/hotchoc. We can get the current drink via the existing consumables framework, so we should just need to check if it's coffee/hotchoc.

**Review by multimokia — CHANGES_REQUESTED** [review]

File: Monika After Story/game/script-topics.rpy (hunk 1 above)
Comment (lunulae) [reply Monika After Story/game/script-topics.rpy:16322]: I'm not sure how to code this, but I'd be happy to write variant dialogue for each case if we wanted to do this 👍

Comment (Booplicate) [reply Monika After Story/game/script-topics.rpy:16322]: You can get current drink via `MASConsumable._getCurrentDrink()`, and food via `MASConsumable._getCurrentFood()`. They'd return `None` if Monika doesn't drink/eat anything. For example, to compare it with coffee, you'd need to do this `MASConsumable._getCurrentDrink() is mas_getConsumable("coffee")`.

**Comment by lunulae** [comment]
I put up some alternate paths for if she's drinking coffee, but I'm a little worried the coffee path sounds a bit repetitive. Might make some tweaks before final release.

**Comment by jmwall24** [comment]
Well, we kinda missed the V-Day cut-off for this (I'll be honest, this went completely under my radar) but better late than never.

**Review by jmwall24 — DISMISSED** [review]

**Review by multimokia — APPROVED** [review]

**Review by multimokia — DISMISSED** [review]

**Review by Booplicate — APPROVED** [review]

**Review by multimokia — APPROVED** [review]

**Review by jmwall24 — APPROVED** [review]

**Pull request merged**